src/java/com/sarality/app/view/BaseViewInitializer.java

```diff
@@ -26,13 +26,13 @@
  *
  * @param <V> Type of view being initialized.
  * @param <T> Type of data needed to initialize the view.
- *
  * @author abhideep@ (Abhideep Singh)
  */
 public abstract class BaseViewInitializer<V extends View, T> implements ViewInitializer<V, T>, DataConsumer<T> {
 
   private static final Logger logger = LoggerFactory.getLogger(BaseViewInitializer.class);
 
+  private static int loader = 0;
```

Comment:
i will check this in the interim because the initialisers are just not working right now.